### PR TITLE
gh-20 Repository re-organization

### PR DESCRIPTION
**Pull request by Andrii77711**
https://github.com/Andrii77711/order-management-service/issues/20

Please merge after gh-7

### CODE REVIEW

File: commons-entity/pom.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    </parent>
+
+    <artifactId>commons-entity</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
```
Comment (taras-oglabyak) [inline commons-entity/pom.xml]: please check is these properties are needed

File: order-management-service-app/pom.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    </parent>
+
+    <artifactId>order-management-service-app</artifactId>
+    <packaging>jar</packaging>
+
+    <properties>
+        <maven.compiler.source>21</maven.compiler.source>
+        <maven.compiler.target>21</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
```
Comment (taras-oglabyak) [inline order-management-service-app/pom.xml]: please check is these properties are needed

File: order-management-service-app/pom.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+    <build>
```
Comment (taras-oglabyak) [inline order-management-service-app/pom.xml]: jus a note: in general, the `build` sections goes after `dependencies`

**Review by taras-oglabyak — CHANGES_REQUESTED** [review]

File: commons-entity/pom.xml (hunk 1 above)
Comment (Andrii77711) [reply commons-entity/pom.xml]: it is not  needed, so I remove this

File: order-management-service-app/pom.xml (hunk 2 above)
Comment (Andrii77711) [reply order-management-service-app/pom.xml]: it is not  needed, so I remove this

File: order-management-service-app/pom.xml (hunk 3 above)
Comment (Andrii77711) [reply order-management-service-app/pom.xml]: changed

**Review by taras-oglabyak — APPROVED** [review]

File: order-management-service-app/pom.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            <version>0.0.1-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <pluginManagement>
```
Comment (taras-oglabyak) [inline order-management-service-app/pom.xml]: what this `pluginManagement`  does?

Comment (Andrii77711) [reply order-management-service-app/pom.xml]: the <pluginManagement> section in a pom.xml file in Maven is used to define plugin configurations that will be inherited by all child projects or modules in a multi-module project. It is left after refactoring modules. So I forgot remove this, my bad

**Review by taras-oglabyak — APPROVED** [review]

**Pull request merged**